resources/devices/mobile-phone/samsung.json

```diff
@@ -1281,7 +1281,7 @@
             "Device_Name": "Galaxy Note II",
             "Device_Maker": "Samsung",
             "Device_Brand_Name": "Samsung",
-            "Device_Pointing_Method": "unknown",
```

Comment:
Looked this one up as well, also touchscreen: http://www.gsmchoice.com/en/catalogue/samsung/galaxynoteiin7108/Samsung-Galaxy-Note-II-N7108.html